src/components/textarea/TextArea.js

```diff
@@ -133,28 +133,16 @@ export default class TextAreaComponent extends TextFieldComponent {
           if (!settings) {
             settings = {};
           }
-          settings.mode = this.component.as || 'javascript';
+          settings.mode = this.component.as;
           this.addAce(element, settings, (newValue) => this.updateEditorValue(index, newValue)).then((ace) => {
             this.editors[index] = ace;
-            ace.on('input', () => this.acePlaceholder());
```

Comment:
`input` event doesn't actually exist, but 'change' does exist and work
`acePlaceholder` function doesn't exist in `TextAreaComponent`